cmd/manager/main.go

```diff
@@ -61,6 +61,7 @@ func main() {
 	var enforceFirstBoot bool
 	var enforcePowerOff bool
 	var serverResyncInterval time.Duration
+	var claimResyncInterval time.Duration
```

Comment:
Do we want to call this claim resync interal? (as it is only used for the registry sync interval now)

Reply:
hm, we actually don't use it anymore. I will delete this and we can revisit if there is a need again.